misc/python/materialize/cli/ci_closed_issues_detect.py

```diff
@@ -154,8 +158,10 @@ def detect_closed_issues(filename: str) -> list[IssueRef]:
                 assert len(groups) == 1, f"Expected only 1 element in {groups}"
                 group, issue_id = groups[0]
 
+                is_referenced_with_url = "issues/" in issue_match.group(0)
+
                 # Explain plans can look like issue references
-                if int(issue_id) < 10:
```

Comment:
Increased in order not to capture:

```
# TODO(mgree): Make this reproducible
# query T multiline
# EXPLAIN WITH(join_impls) SELECT * FROM t JOIN t2 ON t.x = t2.x JOIN t3 ON t.x = t3.x JOIN t4 ON t.x = t4.x JOIN t5 ON t.x = t5.x JOIN t6 ON t.x = t6.x JOIN t7 ON t.x = t7.x JOIN t8 ON t.x = t8.x JOIN t9 ON t.x = t9.x JOIN t10 ON t.x = t10.x;
# ----
# Explained Query:
#   Project (#0, #1, #0, #3, #0, #5, #0, #7, #0, #9, #0, #11, #0, #13, #0, #15, #0, #17, #0, #19)
```

and

```
## TODO: support inserts and updates for named composite type fields
[... many lines ...]
#select * from quadtable;
#----
#1 ("(3.3,4.4)","(5.5,6.6)")
#2 ("(12,4.4)","(5.5,6.6)"
#44 ("(55,)","(,66)")
```

Alternatively, we can decrease the number of comment lines considered after a `TODO`.